<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" tests="6" skipped="0" failures="0" errors="0" timestamp="2021-04-16T01:49:28" hostname="DESKTOP-3R4DVF7" time="0.293">
  <properties/>
  <testcase name="clearFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.142"/>
  <testcase name="getFailPutExceptionFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.029"/>
  <testcase name="evictFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.03"/>
  <testcase name="putFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.031"/>
  <testcase name="getFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.029"/>
  <testcase name="getPutNewElementFail" classname="org.springframework.cache.jcache.interceptor.JCacheErrorHandlerTests" time="0.032"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
